tests/framework/Optimizers/RavenRunningRaven/innerRun/parabola.py

```diff
@@ -0,0 +1,32 @@
+#***************************************
+#* Simple analytic test ExternalModule *
+#***************************************
+#
```

Comment:
can't we just point to the model in /tests/framework/analyticalModels?

Reply:
If it were the same model, sure, but it's not.